### PR TITLE
prepare v0.1.0 and change License: GPLv3

### LINKED ISSUE #46: Add official license file
In some files of the extension the MIT License is mentioned. It would be good to have a License file in the root folder of the repo.

### PR DESCRIPTION
**Pull request by tintinweb**
👉  https://choosealicense.com/licenses/gpl-3.0/. - fixes #46

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#
+# This does not necessarily list everyone who has contributed code,
+# especially since many employees of one corporation may be contributing.
+# To see the full list of contributors, see the revision history in
+# source control.
+#
+# Format: <name> [(url1) (urlN)]
+#
+tintinweb (http://github.com/tintinweb)
+bigbug (http://github.com/bigbug)
```
Comment (tintinweb) [inline AUTHORS:11]: @bigbug feel free to add more ext. links here if you want 🙏

Comment (bigbug) [reply AUTHORS:11]: Seems sufficient for me :)

**Review by bigbug — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -59,6 +59,13 @@ let args = {
 vscode.commands.executeCommand("graphviz-interactive-preview.preview.beside", args)
 ```
 
+## Maintainers
```
Comment (bigbug) [inline README.md:62]: The description for the search API needs updates.

File: README.md [hunk 3]
```diff
@@ -59,6 +59,13 @@ let args = {
...
+
```
Comment (bigbug) [inline README.md:63]: The gifs which show the extension should be updated

**Review by bigbug — CHANGES_REQUESTED** [review]
Still has to be finalised

File: README.md (hunk 2 above)
Comment (tintinweb) [reply README.md:65]: added `search` field.

File: README.md (hunk 3 above)
Comment (tintinweb) [reply README.md:78]: updated the readme: preview 👉 https://github.com/tintinweb/vscode-interactive-graphviz/blob/f7f1d2fdcb9d4f46d8f3095b017feb9286a33698/README.md

wdyt @bigbug ?

**Pull request merged**